engines/stark/ui/gamewindow.cpp

```diff
@@ -51,6 +53,19 @@ GameWindow::GameWindow(Gfx::Driver *gfx, Cursor *cursor, ActionMenu *actionMenu,
 	_position = Common::Rect(Gfx::Driver::kGameViewportWidth, Gfx::Driver::kGameViewportHeight);
 	_position.translate(0, Gfx::Driver::kTopBorderHeight);
 	_visible = true;
+
+
+	_fadeSurface = new Graphics::Surface();
```

Comment:
I would prefer if the black rect was rendered using a non textured rectangle, or at least using a single pixel texture. So that we don't need to keep a full size surface + texture in memory and upload it to GPU memory on each frame.